src/main/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueActor.scala

```diff
@@ -259,7 +259,7 @@ private[impl] class LaunchQueueActor(
         if (instancesToSchedule.nonEmpty) {
           val scheduled = await(instanceTracker.schedule(instancesToSchedule))
         }
-        logger.info(s"Scheduling (${instancesToSchedule.length}) new instances (first five: ${instancesToSchedule.take(5)} ) due to LaunchQueue.Add for ${runSpec.}")
+        logger.info(s"Scheduling (${instancesToSchedule.length}) new instances (first five: ${instancesToSchedule.take(5)} ) due to LaunchQueue.Add for ${runSpec.id}")
```

Comment:
Don't `take(5)`, print them all. Increases the debuggability a lot.

Reply:
what if the list is reeeeaaaally long? 🤔